static/js/components/widgets/MarkdownEditor.tsx

```diff
@@ -50,7 +50,16 @@ export default function MarkdownEditor(props: Props): JSX.Element {
   const editor = useRef<Editor>()
   const onReady = useCallback((editorInstance: Editor) => {
     editor.current = editorInstance
-    if (process.env.NODE_ENV === "development" && editor.current) {
+    if (!editor.current) {
+      /**
+       * It is unclear to me why this happens.
+       * It seems like when our MarkdownEditor opens, an editor is created,
+       * immediately destroyed, onReady is called (with null), and then
+       * re-created, and onReady is called again (with real editor)
+       */
+      return
+    }
+    if (process.env.NODE_ENV === "development") {
       CKEditorInspector.attach(editor)
     }
     editor.current.model.schema.addAttributeCheck(checkNotSubAndSup)
```

Comment:
We've had a mysterious `TypeError: Cannot read properties of null (reading 'destroy')` console.error for a long time (the second console.error below).

I've never really understood the cause of it, but it has never caused problems that I am aware of. This line of code introduced a similar error (presumably related, since they are always logged together as far as I can tell). I'd like to avoid this since, unlike the other error, this really is coming from our code.

<img width="815" alt="Screenshot 2022-12-21 at 2 05 17 PM" src="https://user-images.githubusercontent.com/9010790/208986293-1afd2bb0-8bba-41a4-8f7e-745dc21ff5fb.png">